docs/src/SurfaceFluxes.md

```diff
@@ -141,3 +141,79 @@ u(z) = u_{\star} \frac{F_m(z,z_{0m}, L_O)}{\kappa} + u_{sfc},
+
+Note that CLM4.5 also uses MOST, and its canopy layer ignores the direct effects of turbulence, parameterizing $u$ as $u_*$. This is suboptimal. See \cite{Bonan2018} for more details. (CLM4.5 uses a parameterisation that assumes that the wind speed within the canopy is equal to the friction velocity $u_{\star}$.)
+
+#### Modified MOST in RSL
+Following \cite{Physick1995}, we can provide extensions to the MOST to capture the flow behaviour within the roughness-sublayer whose maximum height is denoted by $z_{*}$. Equation (2) in \cite{Physick1995} corresponds to (\ref{eq:F_h_fin-diff}) in this document, and gives the velocity profiles in the surface layer following the canonical MOST. For model levels $z$ such that $d \leq z \leq z_{\star}$ (within the RSL),  \cite{Physick1995} suggest the following modification to the velocity gradient profile. The profile would be:
+
+```math 
+\begin{equation}
+    \frac{\partial u (z-d)}{\partial z} = (u_{\star}/\kappa z) \Phi_M(\zeta)\phi_M(z-d/z_{\star}-d), 
+\end{equation}
+```
+
+where $\Phi_M$ is the stability function applied in the canonical MOST theory, 
+```math
+\begin{equation}
+    \phi_M\Big(\frac{z-d}{z_{\star}-d}\Big) = 0.5\exp(\ln(2)\frac{z-d}{z_{\star}-d}),
+\end{equation}
+```
+and is estimated from flux-tower data over flat, tree-covered terrain with $z_{0}$ ranging from $0.4-0.9 ~\mathrm{m}$. The stability parameter is defined by $\zeta = (z-d)/L$ following the zero-plane displacement correction.
+
+Thus, for recovery profiles defined in \ref{eq:vel_ref_profile}, the modified expression incorporating the RSL model results in 
+```math
+\begin{equation}
+    u(z-d) = \frac{u_{\star}}{\kappa} \Big(F_{m} + \int_{z_{0}}^{z}  \Phi_M(1-\phi_M)(z-d)^{-1} \,dz\Big) + u(z_{0}), 
+\end{equation} 
+```
+with non-zero $u(z_{0})$. This can also be expressed as 
+```math
+\begin{equation}
+    u(z-d) = \frac{u_{\star}}{\kappa} \Big(F_{m} + \int_{z}^{z_{\star}}  \Phi_M(1-\phi_M)(z-d)^{-1} \,dz\Big). 
+\end{equation} for $z \leq z_{\star}$. 
+```
+While \cite{Physick1995} comment on the application of surface-layer functions defined by \cite{Businger1971}, we can generalise to the family of similarity functions provided provided in the SurfaceFluxes.jl package. 
+
+Note that $K_m$ is also modified by $\phi_M$ if model levels reach the RSL. 
+
+The authors assume that the deep canopy layer (between $z=0$ and $z=d$) has no storage capacity and it gives off the same fluxes it receives from the soil. They further assume that the surface layer height $h_s = 0.04 z_{i}$, where $z_{i}$ is the height of the planetary boundary layer (PBL).   
+#### Modified MOST in RSL and exponential canopy profiles coupled}
```

Comment:
Extra }

Also, shall be comment that this and the Wang&Cionco methods are not currently implemented?

Reply:
Yes : I'll add a comment here